src/styles/blocks/conatiner.scss

```
@@ -0,0 +1,5 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(autofill, 1fr);
```

```suggestion
  grid-template-columns: repeat(auto-fill, 1fr);
```
You can't use autofill with fr

---

in your solution it's redundant